ssm/variational.py

```diff
@@ -255,8 +255,13 @@ def _initialize_variational_params(self, data, input, mask, tag):
         # of the emissions invert function.
         J_diag = np.tile(1.0 / self.initial_variance * np.eye(D)[None, :, :], (T, 1, 1))
         J_lower_diag = np.zeros((T-1, D, D))
-        h = (1.0 / self.initial_variance) \
-            * self.model.emissions.invert(data, input=input, mask=mask, tag=tag)
+        if self.model.emissions.single_subspace:
+            h = (1.0 / self.initial_variance) \
+                * self.model.emissions.invert(data, input=input, mask=mask, tag=tag)
+        else:
+            # TODO smarter inversion with multiple subspace!
```

Comment:
let's issue a warning for the multiple subspace initialization

Reply:
I added a warning that the posterior initialization has not been implemented for multiple subspaces.